src/components/Content.jsx

```diff
@@ -2,16 +2,15 @@ import { Route, Routes } from 'react-router-dom';
 import { ArticleList } from '../pages/ArticleList';
 import { MostRecentArticles } from '../pages/MostRecentArticles';
 import { useEffect, useState } from 'react';
+import { getAllArticles } from '../utils/api';
 
 export const Content = () => {
   const [articleList, setArticleList] = useState([]);
```

Comment:
some loading state would be excellent - just in case the internet connection is slow. It will enhance the user experience 😄